### ISSUE TITLE
Sceneform support GLB and GLTF for animated models

### ISSUE THREAD

**Issue by sergeevsnaappy** [google-ar/sceneform-android-sdk#923]
Hello! Are there any news on animated models support for GLB and GLTF? 
Conversion to SFB is a huge pain in the back - it takes great amount of resources with little reward. A lot of objects are not converted properly because of textures problems and some are crashing the app even after successful conversion.

**Comment by gauravbodra** [comment]
Yes, I too had similar problem. I want to covert obj+mtl+textures in SFB file format but it is time and resource consuming.

**Comment by House3D** [comment]
1 vote for this support.
